sdk/servicebus/azure-messaging-servicebus/src/test/java/com/azure/messaging/servicebus/IntegrationTestBase.java

```diff
@@ -253,20 +253,32 @@ protected ServiceBusClientBuilder getBuilder(boolean useCredentials) {
 
     protected ServiceBusSenderClientBuilder getSenderBuilder(boolean useCredentials, MessagingEntityType entityType,
         int entityIndex, boolean isSessionAware, boolean sharedConnection) {
+        return getSenderBuilder(useCredentials, entityType, entityIndex, -1, isSessionAware, sharedConnection);
```

Comment:
Do we need to have this in the integration test base? How many test classes actually need the viaQueueName test case? This method returns a builder, you can set that name for the particular test case.